src/pages/billing/List.js

```diff
@@ -329,6 +314,6 @@ export const addNavPaths = () => {
   Nav.defPath('billing', {
     path: '/billing',
     component: BillingList,
-    title: 'Billing Management'
+    title: ({ selectedName }) => `Billing ${selectedName ? `- ${selectedName}` : 'Management'}`
```

Comment:
This isn't working, I think the parameter should be destructured as `{ queryParams: { selectedName } }`

Reply:
Destructuring the parameter does not update the title when I select a different billing account, because the title does not update to the new `selectedName`. 

What if I put the title update into `componentDidUpdate()`:
```
document.title = `Billing ${selectedName ? `- ${selectedName}` : 'Management'}`
```
This does update the title when I change the selectedName. Is there a cost to doing it this way?